### ISSUE TITLE
Can build appimage but it fails to run

### ISSUE THREAD

**Issue by ChrisOfBristol** [AppImage/AppImageKit#1111]
I just get this error: 
execv error: : No such file or directory

**Comment by azubieta** [comment]
Hello @ChrisOfBristol,
Could you please provide more information about how are you building the AppImage? We will need to know in which system are you building the AppImage, which tools are you using and what kind of application.

**Comment by ChrisOfBristol** [comment]
I should have thought of that...I found an image I had taken from a YouTube video ages ago and followed the instructions on that. More details in a moment.
![appimagefromdeb](https://user-images.githubusercontent.com/20310498/104520759-9cbd5600-55f3-11eb-85c1-96c2cf907b69.png)

**Comment by azubieta** [comment]
Ok, AppImageAssistan is deprecated.
If you want to convert a binary package into an AppImage please check: https://docs.appimage.org/packaging-guide/converting-binary-packages/index.html

If you want to pack from sources check: https://docs.appimage.org/packaging-guide/from-source/index.html

**Comment by azubieta** [comment]
Closing as it refers to an AppImageAssistant issue which is deprecated.
Please feel free to reopen if required.

**Comment by ChrisOfBristol** [comment]
[I posted on Discourse originally](https://discourse.appimage.org/t/flatpak-appimage/2223) that I wanted to create an AppImage from a Flatpak or Flatpak manifest, but there was no answer to that, then I found this way which I hoped would convert from a .deb. The way you have linked to looks like it requires the writing of a manifest, which I imagine has a different format to a Flatpak manifest. I agree with this principle from the home page of your website “…without having to build for gazillions of different distributions.” and in a similar way am reluctant to learn how to create and build from another type of manifest. So if it neither converting a deb or a Flatpak works I'll manage with those.
